### PR TITLE
JLab: add toolbar and refresh button for the voila preview

### PR DESCRIPTION
**Pull request by jtpio**
Add the toolbar and a refresh button for the voila preview.

image

### CODE REVIEW

**Comment by jtpio** [comment]
cc @aschlaep

**Comment by wolfv** [comment]
i think pressing the voila button could also automatically save the notebook state ... or open a modal window?

File: packages/jupyterlab-voila/src/preview.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    let { url, label, ...opts } = options;
+    super({
+      ...opts,
+      content: new IFrame()
+    });
+
+    this.content.url = url;
+    this.content.title.label = label;
+    this.content.title.icon = VOILA_ICON_CLASS;
+    this.content.id = `voila-${++Private.count}`;
```
Comment (martinRenou) [inline packages/jupyterlab-voila/src/preview.ts]: What is it for this `id`?

Comment (jtpio) [reply packages/jupyterlab-voila/src/preview.ts]: It's used as the DOM id for the phosphor widget

**Comment by jtpio** [comment]
> or open a modal window?

A modal window for saving the notebook?

**Comment by SylvainCorlay** [comment]
As discussed in-person. Merging for the patch release.

**Comment by SylvainCorlay** [comment]
We will iterate on this with

- new buttons to select templates, and template parameters
- more logic around the restarting / saving.

**Pull request merged**